index.html

```diff
@@ -189,9 +193,11 @@ <h2>Partners</h2>
         </ul>
       </div>
       <div class="foot">
-        <div class="footer-logo">Logo from canva</div>
+        <div class="footer-logo">
+          <img src="./images/conference-logo.gif" alt="BMEN Global Logo" />
+        </div>
         <p class="copyright">
-          2015 Creative Commons Korea <br />
+          2022 BMEN Global Conference <br />
           Some Rights Reserved.
         </p>
       </div>
```

Comment:
- [ ]  Kindly give your footer a dark background to match the original design

### Your footer

![image](https://user-images.githubusercontent.com/98453979/190693189-e1ef7c74-31ab-4814-81ef-e14d2f57b6d3.png)

### Original Design
![image](https://user-images.githubusercontent.com/98453979/190693329-b99ccacd-b394-4a1e-808b-d60a58fc2308.png)

Reply:
Thanks for the suggestion. I will make that change immediately